### ISSUE TITLE
Community engagement: Study Groups, Clubs & cohorts 

### ISSUE THREAD

**Issue by kaythaney** [MozillaFoundation/MoFoLondon#8]
## Description:

We'll be digging into onboarding plans for both programs, infrastructure needs for community-driven events + resource creation, and looking at opportunities to connect across program / region. 
### Time requirements:

1-2h.
### Space requirements:

Flipcharts + sticky notes.
### Required attendees:

MSL: @auremoser , @zee-moz , @arlissc + 2 MSL volunteers (Anna + Rob)

Clubs: @amirad, Caro, Julia, @soapdog  
### Optional attendees:

Open to others across MLN, MoFo, + Participation

**Comment by OpenMatt** [comment]
@kaythaney do you have a preference in terms of which day you do this?

**Comment by ldecoursy** [comment]
@hannahkane

**Comment by hannahkane** [comment]
@kaythaney - is this the one where you'd like for representatives from the Production team to show up?

**Comment by kaythaney** [comment]
the study group onboarding session will likely get more into curriculum delivery + design needs. this one could benefit from someone who could help dig into the infrastructure part for clubs/study group reporting, event management, training delivery. :)

sent on the move. apologies for typos.

**Comment by hannahkane** [comment]
I'd love for @sabrinang @flukeout and @Pomax to attend this session then. They make up the current "Organizing" team for MLN production, and this work falls squarely into that bucket. Will see if we can schedule it so that some of us can attend.

**Comment by soapdog** [comment]
Should we have someone who has been recently onboarded at this session to help us with what people need from their point of view?

**Comment by ldecoursy** [comment]
@auremoser @zee-moz @arlissc In Kaitlin's absence, are you able to answer the above q's? Do you know if this is scoped out somewhere?  

It's currently in the schedule for Wednesday at 1 for 90 minutes.

**Comment by auremoser** [comment]
I can try: 
- @hannahkane 's suggested attendees are welcome (cc @sabrinang , @flukeout , @Pomax ) but not obliged.
- @soapdog we have 2 community members joining us (Anna + Rob) mentioned in the original request, one runs a study group and the other is a research member of our Open Leadership Cohort, so I think that's some newbie perspective, but if you can think of someone else you'd like to see there I'm all ears. :)

I'm not sure it's officially scoped but I think we'll put together an agenda and notes after the Global Sprint (June 2-3). Thanks!

**Comment by ldecoursy** [comment]
@amirad @jvallera Are there RCs attending that should join this?

**Comment by amirad** [comment]
Does this happen after our clubs deep dive? We will be covering many onboarding and infastructure requirements in our clubs session.

**Comment by ldecoursy** [comment]
It does. Clubs deep dive, then lunch, then this...  https://docs.google.com/spreadsheets/d/19lBkFBuPecVtD5IuXefbaxm-13vpfZknuXWlF3gYI-0/edit#gid=512500479

**Comment by amirad** [comment]
Wonderbar. We can do that and share some of our discussion here. Do you know who is running/organizing this meeting? Is that you @kaythaney ? Would love to sync up on objectives!

**Comment by kaythaney** [comment]
Would love for @auremoser + @amirad  to lead this, with @zee-moz supporting to speak to onboarding work for our side (maybe Caro or Julia on the Clubs side). 

@auremoser + @amirad - let me know if co-leading is OK with both of you!

**Comment by amirad** [comment]
WFM

**Comment by ldecoursy** [comment]
@mcdermott415 probab good for some Hivers to attend too?

**Comment by auremoser** [comment]
@amirad @zee-moz @kaythaney I put together an agenda with sections for some basics on the Study Groups, OLC. Feel free to fill in the Clubs section @amirad (line 36) , and add whatever agenda items you think are appropriate (line 32). 

https://public.etherpad-mozilla.org/p/mofo-communities

**Comment by lauradereynal** [comment]
@kaythaney Im interested to attend as well. 
We have a looooot of community work with our project in Kenya and this seems like a cross-org collaboration opportunity 

cc @secretrobotron

**Comment by mcdermott415** [comment]
Thanks Lainie. I'll let folks know about the session. M

**Comment by auremoser** [comment]
Yeah @LauraReynal ! All welcome. Also, interested in connecting with you on Kenya as we'll be there late July.

**Comment by amirad** [comment]
Thanks @auremoser - added notes to the pad but will ping you later to talk through more details of the flow!

**Comment by ldecoursy** [comment]
@mcdermott415 Can you let me know who from Hive might be attending this so I can add them to the schedule app?

**Comment by mcdermott415** [comment]
I don't know - I shared with group as voluntary session, but Hana and
Kenyatta might find value.
